image-capture/update-camera-zoom.js

```diff
@@ -1,26 +1,35 @@
 navigator.mediaDevices.getUserMedia({video: true})
-.then(mediaStream => {
+.then(async mediaStream => {
   document.querySelector('video').srcObject = mediaStream;
 
+  // Needed because getCapabilities may be null if accessed too early...
```

Comment:
Please mention here the bug [crbug.com/711524](https://crbug.com/711524) and perhaps explain that capabilities/settings can only be retrieved after the device starts streaming, and that this happens after and asynchronously w.r.t. `getUserMedia()` return.